src/server.rs

```diff
@@ -262,6 +279,8 @@ fn read<'r>(
     let messages =
         DatabaseManager::read_records(&conn, &user_id, &device_id, &index, &limit).unwrap();
     let mut msg_max: u64 = 0;
+    tags.extra
```

Comment:
Probably similar to above, too much cardinality for a tag? This might be handy as a separate metric itself instead though?

Reply:
Disregard